prowler/providers/gcp/gcp_provider.py

```diff
@@ -249,19 +249,24 @@ def get_projects(self) -> dict[str, GCPProject]:
                 )
 
         except HttpError as http_error:
-
             if http_error.status_code == 403 and "organizations" in http_error.uri:
                 logger.error(
                     f"{http_error.__class__.__name__}[{http_error.__traceback__.tb_lineno}]: {http_error.error_details} to get Organizations data."
                 )
         except Exception as error:
-            logger.error(
-                f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
-            )
-            # TODO: we cannot print this for whatever exception
-            print(
-                f"\n{Fore.YELLOW}Cloud Resource Manager API {Style.RESET_ALL}has not been used before or it is disabled.\nEnable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/ then retry."
-            )
+            if error.__class__.__name__ == "RefreshError":
+                logger.critical(
+                    f"\n{Fore.YELLOW}Google Cloud SDK {Style.RESET_ALL}has not been authenticated or the credentials have expired.\nAuthenticate by running 'gcloud auth application-default login' then retry."
```

Comment:
```suggestion
                    f"Google Cloud SDK has not been authenticated or the credentials have expired. Authenticate by running 'gcloud auth application-default login' then retry."
```

If you want to set a log message please use the above format.